cypress/page/payment-step.page.ts

```diff
@@ -13,7 +16,8 @@ class PaymentStepPage {
     }
 
     public getConfirmationMessage() {
-        cy.get("#center_column > div.box > p > strong").should("have.text", "Your order on My Store is complete.");
+        this.confirmationMessageText = cy.get(this.confirmationMessageTag);
+        return this.confirmationMessageText;
```

Comment:
there is no need of the confirmationMessageText property
```suggestion
        return cy.get(this.confirmationMessageTag);
```